bqetl_project.yaml

```diff
@@ -423,6 +423,8 @@ generate:
       datasets:
       - fenix
       app_ids:
+      - debug-ping-view
+      - glean-dictionary
```

Comment:
Looking at the others I _think_ this should be with underscores as well (this is the "app id turned table name"-id I guess? We seem to be a bit inconsistent with the naming on this one)

Reply:
@badboy good point. I took these ids from probe-scraper's repositories.yaml. Fenix uses dots (`.`) as separators, that I believe get transformed to underscores. OTOH, Debug Ping View and Glean Dictionary app ids there have `-` as separators. @scholtzan which is the correct one?